util/util.go

```diff
@@ -202,13 +202,26 @@ func SliceRemoveIfPresent(list []interface{}, search interface{}) []interface{}
 //   - parameters will include path parameters
 func ParsePath(userSuppliedPath, endpoint string, d *schema.ResourceData) string {
 	fields := strings.Split(endpoint, "/")
-	// The second field should be the one the user supplied rather
+	if fields[0] == "" {
+		// There was a leading slash that should be trimmed.
+		fields = fields[1:]
+	}
+	isAuthPath := false
+	if fields[0] == "auth" {
```

Comment:
The logic in this method needed to be changed to support paths that are prefixed with "auth".